### ISSUE TITLE
Email's don't use current domain

### ISSUE THREAD

**Issue by TkTech** [jointakahe/takahe#55]
The email templates for `account_new.html` and `password_reset.html` are hardcoded to use `MAIN_DOMAIN` instead of the current request domain.

**Comment by andrewgodwin** [comment]
Yes, because they are sent from background workers rather than a view. `MAIN_DOMAIN` is going to have to be something that everyone uses for those links, at minimum, even if we redirect them to the service domain after login.

**Comment by TkTech** [comment]
Ah, I had a PR for this. I was using the domain from the user's first identity which I was considering the "default".

**Comment by andrewgodwin** [comment]
Hmm, interesting - I had presumed that new accounts would not have an identity yet, so the Create Account email would _have_ to go to MAIN_DOMAIN. You're right though, password resets could try their first identity.

**Comment by TkTech** [comment]
Yeah, the process right now is to create the domain, create a user, add an identity, then send them a password reset (since invites aren't there just yet)

**Comment by gwagstaff** [comment]
Grabbing this issue + taking the chance to spruce up the email templates with html
